npc/re/custom/lasagna/lasagna_npcs.txt

```diff
@@ -7691,7 +7691,7 @@ lasagna,298,84,5	script	Vigilante#dorcon01	4_DR_SOLDIER,{
 			mes "[Vigilante]";
 			mes "Come with me then. The <NAVI>^4D4DFF[Dragon's Nest]^000000<INFO>lasa_fild02,340,244,000,1,</INFO></NAVI> is in the cave at the right end of the field outside the village.";
 			erasequest 14548;// Crazy Dragon
-			setquest 14549;// No news is not a good news
+			setquest 14549;// No news is not good news
```

Comment:
If you want to change it, please also fix it in the quest_db.yml
But to me, even the new name is not proper English.
Summon @aleos89

Reply:
In English the saying is "No news is good news" but in this case they are saying that no news is not good. It can be written as the correction is above.